lib/pwork/async/manager.rb

```diff
@@ -0,0 +1,68 @@
+require 'thread'
+
+module PWork
+  module Async
+    class Manager
+      def initialize
+        @queue = Queue.new
+        @threads = []
+        @running = false
+      end
+      # This method is called to determine if this manager is running or not
```

Comment:
This file has weird spacing, I think an empty line between methods would be better.